stress-ng.h

```diff
@@ -69,6 +69,22 @@
 #include <time.h>
 #include <unistd.h>
 
+
+#ifndef SCHED_FLAG_RESET_ON_FORK
+#define SCHED_FLAG_RESET_ON_FORK    0x01
+#endif
+
+/*
+ * SCHED_DEADLINE
+ */
+#ifndef  SCHED_FLAG_RECLAIM
+#define SCHED_FLAG_RECLAIM      0x02
+#endif
+
+#ifndef SCHED_FLAG_DL_OVERRUN
+#define SCHED_FLAG_DL_OVERRUN       0x04
+#endif
+
 /*
```

Comment:
Can these #define values 0x01, 0x02, 0x04 be indented with tabs so they align.  Also, the stress-ng convention is to put all these constants in ( )  too.

Reply:
Sure. will modify in v2.